akka-http-core/src/test/scala/akka/http/scaladsl/GracefulTerminationSpec.scala

```diff
@@ -122,7 +122,7 @@ class GracefulTerminationSpec extends WordSpec with Matchers with BeforeAndAfter
 
     "in-flight request responses should include Connection: close and connection should be closed" in new TestSetup {
       val r1 = makeRequest() // establish connection
-      val time: FiniteDuration = 3.seconds
+      val time: FiniteDuration = 6.seconds
```

Comment:
This is unrelated, revert.

Reply:
This test fails due to lack of time on Jenkins. How can I make the unrelated test to pass otherwise?

Reply:
Then it should fail for all other PR validation and our nightlies on Jenkins as well, which I'm pretty sure it doesn't.

Reply:
No, it should not - it can fail from time to time depending on agent.